Makefile

```diff
@@ -0,0 +1,28 @@
+NIM_BUILD_SCRIPT_URL = https://raw.githubusercontent.com/status-im/nimbus-build-system/master/scripts/build_nim.sh
```

Comment:
let's put this in a shell script instead of makefile, ie jenkins.sh

Reply:
because it's not integrated with nimbus-build-system and thus confusing - also, it doesn't have dependencies set up correctly etc so.. it's more like a shell script really

Reply:
That's fine by me, but just because there's a Makefile doesn't mean it has to be integrated with everything the project does.

Reply:
I moved the `Makefile` to `ci` folder.

Reply:
Or, alternatively, I could just add some basic targets to it in the repo root like `make test` or `make update`.

Reply:
in `ci` it's fine - the "official" way of running the tests is `nimble test` - setting up a nim environment is usually expected to be done already (ie it's not part of the developer workflow), so it shouldn't be part of a top-level makefile regardless - nimbus is special because it's a full app whereas this is just a library.